### DISCUSSION TITLE
Best way to UPDATE without loosing any settings

### DISCUSSION THREAD

**Discussion by Kamikaze01git** [RandomNinjaAtk/docker-radarr-extended#34] [General]
Hello frieds of this docker :)

I disabled autoconfig as i want to use my own settings. And because i dont use PLEX i disabled this script too.
My settings works like a charme for me - all metadatas and trailers are downloaded well.

Now, after a few month, my RADARR tells me that there is an update.
So what would be the best way to update the docker without loosing any settings?

:-)

Thank u in advance

<img width="1364" alt="image" src="https://user-images.githubusercontent.com/80671394/211767300-3f410045-58d9-49c9-bbd2-f3e9c4db9ed9.png">

**Comment by RandomNinjaAtk** [comment]
You update the same way you would update any other docker container... It's not any different...

**Comment by Kamikaze01git** [comment]
Hello.
I'm aware that my question is absolutely basic stuff. But so far I've never had to update a Docker.

My friend "google" [told me](https://mixable.blog/synology-how-do-i-update-an-existing-docker-container-with-a-new-image/) that I just have to download the current image and replace the old one. Settings and data will retain.

But is that true in this case? Do I have to load the radarr-extended or the original radarr docker image (as ther are "only" some extra scripts and the radarr doker remains untouched... and these extra scripts will retain after the update, right?).

It took me a long time until my settings work the way I want (I don't need autoconfig or plex and I'm worried that everything will be activated again after the update... [I only need the trailers and extras]).

So which Docker image should I load? :)

Thank you very much and sorry for my noob question... :-/
I really love this radarr-extended docker !!! It's a GREAT work !! Thank u thousand times for that!!
